### DISCUSSION TITLE
Welcome to next-multilingual Discussions!

### DISCUSSION THREAD

**Discussion by nbouvrette** [Avansai/next-multilingual#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NitrousBGC** [comment]
Hello!

This project is probably the most useful of all translation projects. It has everything we could need to make a perfect SEO-Friendly multilingual website.

I can't understand why others never thought about SEO when creating the projects. I saw no support for sitemaps nor for rel=alternate hreflang.. 

If only I found it earlier.. 😅
I will need to convert a full website from nextI18-next to next-multilingual.
(or would there be a way to only implement the SEO part for the moment?) 

I can't thank this team enough for the wonderful work that has been accomplished.
